comps/lvms/tgi-llava/lvm_tgi.py

```diff
@@ -87,7 +88,14 @@ async def lvm(request: Union[LVMDoc, LVMSearchedMultimodalDoc]) -> Union[TextDoc
         top_k = request.top_k
         top_p = request.top_p
 
-    image = f"data:image/png;base64,{img_b64_str}"
+    if img_b64_str:
+        image = f"data:image/png;base64,{img_b64_str}"
+    else:
+        # Work around an issue where LLaVA-NeXT is not providing good responses when prompted without an image.
+        # Provide an image and then instruct the model to ignore the image.
+        image = "https://raw.githubusercontent.com/opea-project/GenAIExamples/refs/tags/v1.0/AudioQnA/ui/svelte/src/lib/assets/icons/png/audio1.png"
+        prompt = f"Please disregard the image and answer the question. {prompt}"
```

Comment:
Did you compare this with something more authoritative and direct, like "Ignore the image and answer the question"? It will depend on the model's training, and I don't know if it's any better, but it's worth testing if you were still getting references to the image with this.

Reply:
Yes, I tried different prompts, including "Ignore the image and just answer the question"